csharp/private/runtime.BUILD

```diff
@@ -1,9 +1,14 @@
+load("@rules_cc//cc:defs.bzl", "cc_binary")
+load("@d2l_rules_csharp//csharp:defs.bzl", "dotnet_wrapper")
+
 exports_files(
-    glob([
-        "dotnet",
-        "dotnet.exe",  # windows, yeesh
-    ], allow_empty = True) +
-    glob([
+    glob(
+        [
+            "dotnet",
+            "dotnet.exe",  # windows, yeesh
+        ],
+        allow_empty = True,
+    ) + glob([
```

Comment:
What's this change about?

Reply:
This is due to buildifier changing the formatting of this file. I verified that this changed on both my branch, and running on master for this file. In both cases, the output result is as above.